www/js/app.js

```diff
@@ -85,6 +85,9 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
     var globalDropZone = document.getElementById('search-article');
     var configDropZone = document.getElementById('configuration');
     
+    // Title for current url.
+    var curTitle = "A/index";
```

Comment:
@zengchu2 Thank you for this PR. I haven't tested yet, but can I just clarify why you assume here that the current URL is named `A/index`? Is this just a placeholder with no significance that will be overwritten, in which case, should it just be `curTitle = ""`?

Reply:
I hard coded it to be **"A/index"** to enable **readArticle()** load the index page. (In the case of **curTitle = ""**,  the index page will not be loaded due to the early return)

Reply:
I understand the purpose, but you should not hard-code the name of the index page (because it depends on the ZIM file).
A better way to do that would be to set this variable in `goToMainArticle` function (before calling `readArticle`) with the content of `dirEntry` variable.
And the same thing should be done in `goToRandomArticle`.

There also seems to be a confusion between the URL (with or without namespace) and the title of an article (it's my fault and the code is not always clear on that).
When I was referring to a title in #89 , I should have referred to the URL (with namespace, so that it is unique).
It's already what you're putting in the variable, but you should rename the variable to something more meaningful, to indicate that it's the URL we expect to see appearing. Maybe `futureArticleURL` or `expectedArticleURLToBeDisplayed`? @Jaifroid is the master of wording. And an explanation might be put above on what is our intention (which problem we are solving with that)